billboard.py

```diff
@@ -69,9 +70,10 @@ class ChartEntry:
         isNew: Whether the track is new to the chart, as a boolean.
     """
 
-    def __init__(self, title, artist, peakPos, lastPos, weeks, rank, isNew):
+    def __init__(self, title, artist, cover, peakPos, lastPos, weeks, rank, isNew):
         self.title = title
         self.artist = artist
+        self.cover = cover
```

Comment:
Can we call this `image`? It looks like they're not always album covers.

Reply:
Updated the codebase